invenio_deposit/views/ui.py

```diff
@@ -68,12 +76,37 @@ def index():
         return render_template(current_app.config['DEPOSIT_UI_INDEX_TEMPLATE'])
 
     @blueprint.route('/deposit/new')
+    @blueprint.route('/deposit/new/<string:deposit_type>')
```

Comment:
It would be helpful to have the opportunity to also customize if the `/deposit/new` endpoint will be created.. So, even if someone wants one he can create his own, as we do https://github.com/cernanalysispreservation/analysis-preservation.cern.ch/blob/master/cap/modules/deposit/views/ui.py#L88